tests/tinygraph-sanity.c

```diff
@@ -7,56 +7,59 @@ static int _testNum;
 int main(int argc, char *argv[])
 {
     int BFSsize, i, j, n, testNum;
-    TINY_GRAPH *G, *Gbar, *GG;
+    TINY_GRAPH *G, *Complete, *GG, *GGC, *U,*Gbar, *U2;
     srand48(time(NULL));
     for(testNum=0;testNum<10000;testNum++)
     {
 	n = 6 + lrand48() % 3; // ensure G->n is at least 2
-	if(drand48() < 0.5)
-	    G = TinyGraphAlloc(n);
-	else
-	    G = TinyGraphSelfAlloc(n);
-
+	G = TinyGraphAlloc(n,(lrand48()%2==0),(lrand48()%2==0));
```

Comment:
LOL. Nice move with the lrand % 2... though it's technically not rigorously correct. Ask me to explain sometime how to use random numbers in a rigorously correct way.